Resources/Prototypes/Entities/Clothing/Eyes/hud.yml

```
@@ -223,6 +223,7 @@
     sprite: Clothing/Eyes/Hud/syndagent.rsi
   - type: ShowSyndicateIcons
   - type: ShowHealthBars
+  - type: SolutionScanner #Sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)